doc/manual/preprocessing/ssp.rst

```diff
@@ -126,3 +115,77 @@ decomposition, and employ the eigenvectors corresponding to the
 highest eigenvalues as basis for the noise subspace. It is also
 customary to use a separate set of vectors for magnetometers and
 gradiometers in the Vectorview system.
+
+Average reference
+-----------------
+
+The EEG average reference is the mean signal over all the sensors. It is typical in EEG analysis to subtract the average reference from all the sensor signals :math:`b^{1}(t), ..., b^{n}(t)`. That is:
+
+.. math::	{b}^{j}_{s}(t) = b^{j}(t) - \frac{1}{n}\sum_{k}{b^k(t)}
+   :label: eeg_proj
+
+where the noise term :math:`b_{n}^{j}(t)` is given by
+
+.. math:: 	b_{n}^{j}(t) = \frac{1}{n}\sum_{k}{b^k(t)}
+   :label: noise_term
+
+Thus, the projector vector :math:`P_{\perp}` will be given by :math:`P_{\perp}=\frac{1}{n}[1, 1, ..., 1]`
```

Comment:
I would add a pitfall here. Basically you can remove signal too when applying SSPs. So it's usually
a good idea to check how much the effect of interest if reduced by applying SSPs. It can happen
that an SSP removes both the artifact and the signal of interest.